dotCMS/src/main/java/com/dotcms/graphql/business/GraphqlAPIImpl.java

```diff
@@ -161,13 +161,10 @@ private void createSchemaType(ContentType contentType,
         // add CONTENT interface fields
         builder.fields(InterfaceType.CONTENTLET.getType().getFieldDefinitions());
 
-        // TODO commented while pending for researching. Do not remove 
-        /*
         if(InterfaceType.getInterfaceForBaseType(contentType.baseType())!=null) {
             builder.withInterface(InterfaceType.getInterfaceForBaseType(contentType.baseType()));
         }
-        */
-        
+
```

Comment:
Restore the Interfaces but now without fields to avoid clients breaking when upgrading and the basetype fields changing